libs/components/src/lib/table/account-table/table2/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { Heading, Text, qtMerge } from '@deriv/quill-design';
+import clsx from 'clsx';
+import { TableContent } from '../types';
+
+export interface AccountTableProps {
+  className?: string;
+  tableData: TableContent;
+  contentClasses?: string;
+}
+
+export const AccountTable2: React.FC<AccountTableProps> = ({
```

Comment:
this component is too huge, please break it down and create more small components.